src/consent.js

```diff
@@ -78,6 +78,23 @@ export function getConsentPolicyInfo(element, policyId) {
   );
 }
 
+/**
+ * @param {!Element|!ShadowRoot} element
+ * @param {string} policyId
+ * @return {!Promise<boolean>}
+ */
+export function getGdprAppliesInfo(element, policyId) {
```

Comment:
Is the plan to migrate to `getConsentPolicyInfo` and deprecate the method here?

Reply:
See comments below

Reply:
`// Return the stored gdpr applies value.` doesn't tell what the future plan is.

Reply:
Oops I was unclear, the plan as [discussed offline](https://github.com/ampproject/amphtml/pull/27759#discussion_r409895533), is to consolidate the methods in src/consent to return a single consent info object, but for now we will keep this method to unblock the ads teams.